packages/core/bootstrap/src/lib/ws/types.ts

```diff
@@ -0,0 +1,30 @@
+
+export interface WSConfig {
+  /** Info about the connection */
+  connectionInfo: {
+    key: string
+  }
+  /** Maximum amount of parallel connections */
+  connectionLimit: number
+  /** Number of ms if no data received this connection is considered dead */
+  connectionTTL: number
+  // TODO: exponential back-off
+  /** Maximum number of connection retries */
+  connectionRetryLimit: number
+  /** Minimum delay between connection retries */
+  connectionRetryDelay: number
+
+  /** Maximum amount of subscriptions per connection */
+  subscriptionLimit: number
```

Comment:
I don't see this used? We should populate the limit from our rate limits package, or it's set manually by the user.

Reply:
#467